tests/unit/modelselection/GradientModelSelection_unittest.cc

```diff
@@ -98,9 +102,9 @@ TEST(GradientModelSelection,select_model_exact_inference)
 	float64_t sigma=lik->get_sigma();
 
 	// compare hyperparameters with result from GPML toolbox
-	EXPECT_NEAR(scale, 0.833180109059697, 1E-2);
-	EXPECT_NEAR(width, 0.190931901479123, 1E-2);
-	EXPECT_NEAR(sigma, 4.25456324418582e-04, 1E-2);
```

Comment:
There are many local modes.
The old result of sigma `EXPECT_NEAR(sigma, 4.25456324418582e-04, 1E-2);` is ignored in the unit test.
The new setting will test all output.